src/text-transform/blocks.ts

```diff
@@ -366,6 +366,33 @@ const config: BlocksConfig = {
             fields: ['description'],
             transformer: yfmTransformer,
         },
+        {
+            fields: ['title'],
+            transformer: typografTransformer,
+            parser: parseTitle,
+        },
+    ],
+    [BlockType.CardLayoutBlock]: [
+        {
+            fields: ['description'],
+            transformer: yfmTransformer,
+        },
+        {
+            fields: ['title'],
+            transformer: typografTransformer,
+            parser: parseTitle,
+        },
+    ],
+    [BlockType.PromoFeaturesBlock]: [
+        {
+            fields: ['description'],
+            transformer: yfmTransformer,
```

Comment:
`BlockHeader` uses `<HTML>` to render description, so in this case description transformed with `yfmTransformer` won't have `yfm` class.